tsconfig.json

```diff
@@ -22,7 +22,8 @@
             "@styles/*": ["./src/styles/*"],
             "@src/*": ["./src/*"],
             "@userActions/*": ["./src/libs/actions/*"]
-        }
+        },
+        "jsx": "react-jsx"
```

Comment:
what does this exactly do?

Reply:
not sure why I didn't get these notifications but I made this change for reasons unrelated directly to this PR, I can remove it. I was running into Typescript issues while working on the issue and this seemed to resolve them. This has to do with how JSX is transformed in TS files.